src/cosalib/meta.py

```diff
@@ -109,6 +191,24 @@ def get(self, *args):
         except KeyError:
             return default
 
+    def get_artifact_meta(self, artifact, unmerged=False):
+        """
+        Return just a dict of the artifact
+        """
+        data = self.dict()
+        dn = os.path.dirname(self.path)
+        alt_path = os.path.join(dn, f"meta.{artifact}.json")
+        if (os.path.exists(alt_path) and unmerged is True and
```

Comment:
`is True` seems a bit redundant, are you trying to avoid a specific case where `unmerged` or `COSA_DELAYED_MERGE` evaluates as to `True` while not being a boolean?

Reply:
Right. We want to make sure its a boolean.